init/hive/init.sh

```diff
@@ -21,7 +21,6 @@
 sed -i '$d' /usr/local/sbin/start.sh
 sed -i '$d' /usr/local/sbin/start.sh
 cp /tmp/hive/core-site.xml /tmp/hadoop-conf
-sed -i "s|hdfs://localhost:9000|hdfs://${HIVE_HOST_IP}:9000|g" /usr/local/hive/conf/hive-site.xml
 /bin/bash /usr/local/sbin/start.sh
```

Comment:
There's placeholder in the configuration file of the hive image: https://github.com/apache/gravitino/blob/main/dev/docker/hive/hive-site.xml

I think it needs to be replaced to `hdfs://hive:9000`

Reply:
I think we don't need to add this line, because in `start.sh`, It has already used `sed` to replace it with `$(hostname)`.

```
sed -i "s/__REPLACE__HOST_NAME/$(hostname)/g" ${HIVE_CONF_DIR}/hive-site.xml
```

https://github.com/apache/gravitino/blob/branch-0.7/dev/docker/hive/start.sh#L37

Reply:
I think we can directly replace `tail -f /dev/null` with empty.
Because maybe `tail -f /dev/null` is not in the last line.

Reply:
I changed it to 

```
sed -i -E 's/tail -f \/dev\/null/\s/g' /usr/local/sbin/start.sh
```

, since last line might be a blank string.